blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,411 @@
+    list-style: none;
+    padding: 0;
+  }
+  
+  .v2-footer .v2-footer-social-media-section {
+    margin-top: 21px;
+    display: flex;
+    gap: 20px;
+    flex-wrap: wrap;
+  }
+  
+  .v2-footer .v2-footer-social-media {
+    border-radius: 17.5px;
+    color: var(--c-tertiary-cool-gray);
+    text-decoration: none;
+    display: flex;
+    width: 30px;
+    height: 30px;
+    align-items: center;
+    justify-content: center;
+    font-size: 20px;
+  }
+  
+  .v2-footer .v2-footer-social-media:hover {
+    color: var(--c-tertiary-cool-white);
+  }
+  
+  .v2-footer .v2-footer-social-media::before {
+    width: 17px;
+  }
+
+  .icon-fb,
+  .icon-twitter,
+  .icon-yt,
+  .icon-instagram,
+  .icon-linkedin {
+    display: flex;
+  }
+
+  
+  .v2-footer h1,
+  .v2-footer h2,
+  .v2footer h3 {
+    font-size: 14px;
+    font-weight: 400;
+    margin-bottom: 5px;
+    border-bottom: 2px solid;
+    line-height: 22px;
+    margin-top: 0;
+  }
+  
+  .v2-footer ul li a,
+  .v2-footer .v2-footer-bottom-section li a {
+    color: var(--c-secondary-silver);
+  }
+  
+  .v2-footer ul li a:hover {
+    color: var(--c-secondary-white);
+    text-decoration: none;
+  }
+  
+  .v2-footer h1 ~ ul,
+  .v2-footer h2 ~ ul,
+  .v2-footer h3 ~ ul {
+    display: flex;
+    flex-flow: row wrap;
+  }
+  
+  .v2-footer h1 ~ ul li,
+  .v2-footer h2 ~ ul li,
+  .v2-footer h3 ~ ul li {
+    width: 50%;
+  }
+  
+  .v2-footer .v2-footer-bottom-section-content {
+    display: flex;
+    align-items: start;
+    font-size: 12px;
+    width: 91%;
+    flex-wrap: wrap;
+    gap: 8px;
+  }
+  
+  .v2-footer .v2-footer-bottom-section li {
+    padding: 10px 0;
+  }
+  
+  .v2-footer .v2-footer-bottom-section li a:hover {
+    color: var(--c-primary-white);
+  }
+  
+  .v2-footer .footer-form-and-social-section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .v2-footer .v2-footer-bottom-section {
+    margin-bottom: 38px;
+    border-top: 1px solid var(--c-tertiary-cool-gray);
+    width: 100%;
+  }
+
+  .v2-footer .v2-footer-links-section {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+  }
+
+  .v2-footer .v2-footer-links-section ul {
+    min-width: 50%;
+  }
+  
+  .v2-footer .v2-footer-form-header h3{
+    font-family: var(--ff-body);
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    letter-spacing: 1.92px;
+    text-transform: uppercase;
+    margin-bottom: 12px;
+  }
+
+  .v2-footer .v2-footer-main-content{
+    display: grid; 
+    grid-template-columns: 1fr; 
+    grid-template-rows: 1fr 1fr 1fr 1fr; 
+    gap: 0 0; 
+    /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+    grid-template-areas: 
+      "v2-logo"
+      "v2-content"
+      "v2-form"
+      "v2-social"; 
+      height: 70vh;
+  }
+  
+
+  .v2-social { 
+    grid-area: v2-social;
+    margin-top: 0px;
+  }
+
+  .v2-content { grid-area: v2-content; }
+
+  .v2-form { 
```

Comment:
The eloqua-form.css file sets `max-width: 950px;`. From around 1024px viewport until the breakpoint, this causes the form to not be aligned to the left. Could you fix that?

Reply:
Updated